pkg/pillar/types/locationconsts.go

```diff
@@ -4,21 +4,62 @@
 package types
 
 const (
-	appImgObj = "appImg.obj"
-
-	// TmpDirname - Temporary dir for zededa components
+	// TmpDirname - temporary dir. for agents to use.
 	TmpDirname = "/var/tmp/zededa"
-	// PersistDir - Persist Directory
+	// AADirname - directory for per-model Assignable Adapters files
+	AADirname = TmpDirname + "/AssignableAdapters"
```

Comment:
This doesn't exist any more (still exists in build but isn't used)

Reply:
Removed it..